lib/js/dom.nim

```diff
@@ -1297,3 +1307,11 @@ proc offsetHeight*(e: Node): int {.importcpp: "#.offsetHeight", nodecl.}
 proc offsetWidth*(e: Node): int {.importcpp: "#.offsetWidth", nodecl.}
 proc offsetTop*(e: Node): int {.importcpp: "#.offsetTop", nodecl.}
 proc offsetLeft*(e: Node): int {.importcpp: "#.offsetLeft", nodecl.}
+
+
+since (1, 3):
+  func newDOMParser*(): DOMParser {.importcpp: "(new DOMParser()​​)".}
+    ## DOM Parser constructor.
+
+  func parseFromString*(this: DOMParser; str: cstring; mimeType = "text/html".cstring): Document {.importcpp.}
```

Comment:
same comment as in your `jsfetch` PR, use an enum since there is a finite set of acceptable values for mimeType, see https://developer.mozilla.org/en-US/docs/Web/API/DOMParser

Reply:
or we can just remove the default `mimeType` and keep it barebones. JS is a wild place and things can change really fast. We should just provide a barebones wrapper and call it a day.